src/shared/clients/codecatalystClient.ts

```diff
@@ -38,24 +38,33 @@ interface CodeCatalystConfig {
     readonly gitHostname: string
 }
 
-export function getCodeCatalystConfig(): CodeCatalystConfig {
-    const stage = DevSettings.instance.get('cawsStage', 'prod')
-
-    if (stage === 'gamma') {
-        return {
-            region: 'us-west-2',
-            endpoint: 'https://public.codecatalyst-gamma.global.api.aws',
-            hostname: 'integ.stage.REMOVED.codes',
-            gitHostname: 'git.gamma.source.caws.REMOVED',
-        }
-    } else {
-        return {
-            region: 'us-east-1',
-            endpoint: 'https://public.codecatalyst.global.api.aws',
-            hostname: 'codecatalyst.aws',
-            gitHostname: 'codecatalyst.aws',
-        }
+export function getCodeCatalystConfig(): Readonly<CodeCatalystConfig> {
+    return getCodeCatalystConfigFromSettings(DevSettings.instance)
+}
+
+export function getCodeCatalystConfigFromSettings(settings: DevSettings): Readonly<CodeCatalystConfig> {
```

Comment:
probably makes sense for this to live in https://github.com/aws/aws-toolkit-vscode/blob/2f7fb0adc426b80ba42207abb93a50a76a90151d/src/shared/settings.ts#L687

that is a common pattern for settings management in the codebase.